apps/qubit/config/qubitConfiguration.class.php

```diff
@@ -31,7 +31,7 @@ public function responseFilterContent(sfEvent $event, $content)
 
     public function listenToChangeCultureEvent(sfEvent $event)
     {
-        setcookie('atom_culture', $event['culture']);
+        setcookie('atom_culture', $event['culture'], ['path' => '/']);
```

Comment:
Maybe this avoids the multiple cookie issue ...

Reply:
Yeah, I'm going to do some more testing to double-check but I think that's the cause of the multiple cookie issue. I hadn't ran into it before.